package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ebury/chameleon-components",
-  "version": "0.1.73",
+  "version": "0.1.74",
```

Comment:
@jaimeperezpedra it is not necessary to commit this if later we'll execute "npm run release" command?

Reply:
I do the release command before to merge the PR
I don't know if this is what you are talking about

Reply:
Ok, got it, You execute separated commands to do the release, thx.